lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -123,7 +123,7 @@ def _check_split_test(self, user_tag):
         content = resp.content
 
         # Assert we see the proper icon in the top display
-        self.assertIn('<a class="{} inactive progress-0 nav-item"'.format(self.ICON_CLASSES[user_tag]), content)
+        self.assertIn('<button class="{} inactive progress-0 nav-item"'.format(self.ICON_CLASSES[user_tag]), content)
```

Comment:
I guess it partly depends on how the html was rendered.  If, for instance, the params came from a python dict, the ordering would be arbitrary but consistent in python 2, but both arbitrary and inconsistent in python 3, where the hash seed is randomized to prevent a certain kind of DOS attack, and then we'd have a flaky test on our hands (if we moved to python 3).  

But that aside, I agree, we can table this for a larger discussion.